packages/react-devtools-shared/src/backend/views/TraceUpdates/canvas.js

```diff
@@ -213,9 +246,18 @@ function initialize(): void {
     position: fixed;
     right: 0;
     top: 0;
-    z-index: 1000000000;
+    background-color: transparent;
+    outline: none;
+    box-shadow: none;
+    border: none;
   `;
 
   const root = window.document.documentElement;
   root.insertBefore(canvas, root.firstChild);
+
+  try {
+    // $FlowFixMe[prop-missing]
+    // $FlowFixMe[incompatible-use]
```

Comment:
Please specify every new FlowFixMe where Popover API is referenced by adding "Flow doesn't recognize Popover API" comment

Reply:
Thanks I've added it to all the necessary places!